flight/Modules/Actuator/actuator.c

```diff
@@ -340,12 +340,23 @@ static void actuatorTask(void* parameters)
 				continue;
 			}
 
-			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO))
+			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO)) {
 				status[ct] = ProcessMixer(ct, curve1, curve2, &mixerSettings, &desired, dT);
-			else
-				status[ct] = -1;
-
 
+				// If a motor type, additional work to be done
+				if(mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) {
+					if (status[ct] > 0) {
+						// Apply curve fitting, mapping the input to the propeller output.
+						status[ct] = actuatorSettings.MotorInputOutputCurveFit[ACTUATORSETTINGS_MOTORINPUTOUTPUTCURVEFIT_A] *
```

Comment:
I was thinking you'd just invert the input and output.. -powf(-x, y) .. granted the prop may not respond exactly the same in reverse depending on geometry of things around it and whether it's a symmetric foil, but it ought to be close.. and beats coming up with some nasty series expansion

> P.S. There's a conversation here about the fact that motors are treated as things that can only run forward, and I think we agree that it's a shame that this assumption is sort of baked into our process.

People are asking me to prioritize fixing that, btw-- make Tau Labs fully 3D and make ground vehicles actually work.  I'm a nav nerd and kinda don't really care about that stuff but I get persuaded to work on acro stuff. ;)  Though I may be working on a vehicle soon that may need negative thrust in some flight regimes for stability.